django_etuovi/etuovi.py

```diff
@@ -0,0 +1,38 @@
+import time
+from ftplib import FTP
+from xml.etree.ElementTree import ElementTree
+
+from django.conf import settings
+from lxml import etree
+
+
+def get_filename():
+    # TODO change this when actual filename confirmed.
+    return "EO_realty2_{}.xml".format(
+        time.strftime("%Y%m%d%H%M%S"),
+    )
+
+
+def send_items(items, transfer_type="all"):
+    root = etree.Element("transferData", version="1.0")
+    group = etree.Element(
+        "transferGroup", type=transfer_type, name=settings.ETUOVI_TRANSFER_ID
+    )
+    root.append(group)
+    for item in items:
+        group.append(item.to_etree())
+    tree = ElementTree(group)
+    filename = get_filename()
+    tree.write(filename, encoding="utf-8", xml_declaration=True)
+
+    session = FTP(
+        host=settings.ETUOVI_FTP_HOST,
+        user=settings.ETUOVI_USER,
+        passwd=settings.ETUOVI_PASSWORD,
+    )
+    # TODO Confirm if the temp filename is required.
+    session.storbinary("{}.temp".format(filename), open(filename, "rb"))
+    session.rename("{}.temp".format(filename), filename)
+    rettrlines = session.retrlines("LIST")
```

Comment:
This was debug to show that the file has been transferred. There is no real use to return the file listing. I did forgot to remove that line from oikotie.
Transfer throws exception if something fails, so I think it is enough?

Reply:
Yeap, let's remove this.